Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -526,8 +526,8 @@ YUI.add('ez-platformuiapp', function (Y) {
             this.get('container').addClass(viewInfo.hideClass);
             if ( viewInfo.instance ) {
                 view = viewInfo.instance;
-                view.set('active', false);
                 viewInfo.serviceInstance.removeTarget(this);
+                view.set('active', false);
```

Comment:
is this change still necessary? to be honest, I'm a bit afraid of the side effects ?